ovssubnet/common.go

```diff
@@ -312,7 +314,12 @@ func (oc *OvsController) StartNode(sync, skipsetup bool) error {
 			log.Errorf("Failed to obtain ContainerNetwork: %v", err)
 			return err
 		}
-		err = oc.flowController.Setup(oc.localSubnet.Sub, containerNetwork)
+		servicesNetwork, err := oc.subnetRegistry.GetServicesNetwork()
```

Comment:
We should only do this in case on multinent solution.
Refer https://github.com/danwinship/openshift-sdn/blob/service-isolation/ovssubnet/common.go#L334